pkg/serverconfig/config.go

```diff
@@ -190,6 +190,10 @@ func addClusterInfo(fs *flag.FlagSet, clusterInfo *ClusterInfo) {
 	if clusterInfo.MasterPublicURL != "" {
 		fs.Set("k8s-public-endpoint", clusterInfo.MasterPublicURL)
 	}
+
+	if clusterInfo.ControlPlaneToplogy != "" {
+		fs.Set("control-plane-topology", string(clusterInfo.ControlPlaneToplogy))
```

Comment:
Would make the flag name more explicit

```suggestion
		fs.Set("control-plane-topology-mode", string(clusterInfo.ControlPlaneToplogy))
```